app/jobs/cleanup_derivatives_job.rb

```diff
@@ -13,6 +13,8 @@ class CleanupDerivativesJob < ApplicationJob
   def perform(file_set_id)
     file_set = query_service.find_by(id: Valkyrie::ID.new(file_set_id))
     Valkyrie::DerivativeService.for(FileSetChangeSet.new(file_set)).cleanup_derivatives
+  rescue Valkyrie::Persistence::ObjectNotFoundError
+    Rails.logger.error "Unable to find FileSet #{file_set_id}"
```

Comment:
I think I would feel confused reading this error message. I can't tell how serious it is; does it mean we've lost data? Or does it just mean the files couldn't be deleted because they were gone already? Since there won't be a stack trace maybe some extra context would be helpful.

Reply:
I think this is happening when we delete FileSets before the CleanupDerivativesJob has a chance to run — see https://github.com/pulibrary/figgy/issues/549#issuecomment-346938857

The ultimate solution to this problem is probably to cleanup derivatives and other files inline before the metadata is deleted.  Or maybe to make the CleanupDerivativesJob not rely on looking up the metadata (by passing the file path instead of the FileSet id?).

Reply:
Cool, okay, maybe we could make it "Unable to find FileSet #{file_set_id} for deletion"? A little extra context would make me feel less nervous if I happened on this message in a log.